api/src/test/java/org/apache/iceberg/expressions/TestInclusiveManifestEvaluator.java

```diff
@@ -347,4 +349,52 @@ public void testStringStartsWith() {
     shouldRead = ManifestEvaluator.forRowFilter(startsWith("some_nulls", "1"), SPEC, false).eval(FILE);
     Assert.assertFalse("Should skip: range doesn't match", shouldRead);
   }
+
+  @Test
+  public void testIntegerIn() {
+    boolean shouldRead = ManifestEvaluator.forRowFilter(in("id", 5, 6), SPEC, true).eval(FILE);
+    Assert.assertFalse("Should not read: id below lower bound", shouldRead);
+
+    shouldRead = ManifestEvaluator.forRowFilter(in("id", 28, 29), SPEC, true).eval(FILE);
+    Assert.assertFalse("Should not read: id below lower bound", shouldRead);
+
+    shouldRead = ManifestEvaluator.forRowFilter(in("id", 30, 31), SPEC, true).eval(FILE);
```

Comment:
nit: It might be more readable if the values 30 and 79 were extracted into min/max variables?

Reply:
Thanks for the comments. I think it is better to put additional info into `Assert` message.
In this way, it will also improve readability for other cases, e.g. values of `5, 6`.